rpcs3/Emu/Cell/lv2/sys_ppu_thread.cpp

```diff
@@ -430,11 +430,11 @@ error_code sys_ppu_thread_rename(u32 thread_id, vm::cptr<char> name)
 	}
 
 	constexpr u32 max_size = 27; // max size including null terminator
+
 	const auto pname = name.get_ptr();
 
-	// thread_ctrl name is not changed (TODO)
-	const std::string res = thread->ppu_name.assign(pname, std::find(pname, pname + max_size, '\0'));
-	sys_ppu_thread.warning(u8"sys_ppu_thread_rename(): Thread renamed to “%s”", res);
+	thread->ppu_tname.store(stx::shared_cptr<std::string>::make(pname, std::find(pname, pname + max_size, '\0')));
+	sys_ppu_thread.warning(u8"sys_ppu_thread_rename(): Thread renamed to “%s”", pname);
```

Comment:
```suggestion
	sys_ppu_thread.warning(u8"sys_ppu_thread_rename(): Thread renamed to “%s”", *thread->ppu_tname.load());
```

Reply:
It was already changed, github is lagging